Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/hyperium/headers"
 authors = ["Sean McArthur <sean@seanmonstar.com>"]
 keywords = ["http", "headers", "hyper", "hyperium"]
 categories = ["web-programming"]
+rust-version = "1.56"
```

Comment:
unfortunately having CI look at this value instead of having to duplicate `minrust: 1.56.0` isn't supported by rust-toolchain action https://github.com/dtolnay/rust-toolchain/issues/70